### PR TITLE
Improve default visibility

### PR DESCRIPTION
**Pull request by mate4st**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #https://github.com/kubermatic/docs/issues/705

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
Add "kubermatic-installer print" command to improve defaulting visibility.
```

### CODE REVIEW

**Comment by kron4eg** [comment]
@WeirdMachine dumping result of yaml.Marshal removes all the comments structs had, so it's not very useful. In the docs directory we have generated files with all the comments and defaults, so we need to use them as a source to those new commands. To embed those files during the build time please use https://pkg.go.dev/embed and https://pkg.go.dev/io/fs.

**Comment by mate4st** [comment]
Yes I like the idea, but to use embed we would need to upgrade the go version to 1.16. So should we leave it for now or introduce some other third party library?

**Comment by kron4eg** [comment]
We already use go 1.16, no “upgrade” is required.

**Comment by mate4st** [comment]
go.mod is still 1.14 or am I missing something here?

**Comment by kron4eg** [comment]
Yes, but we build using 1.16 golabg image. 1.14 been working for us so far since we didn’t used any 1.16 compiler features. Bottom line is: simply bump go.mod version.

File: cmd/kubermatic-installer/cmd_print.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	return handleErrors(logger, setupLogger(logger, func(ctx *cli.Context) error {
+
+		arg := strings.ToLower(ctx.Args().First())
+
+		switch arg {
+		case "":
+			fallthrough
+		case "config":
+			fallthrough
+		case "kubermaticconfiguration":
```
Comment (kron4eg) [inline cmd/kubermatic-installer/cmd_print.go]:
```suggestion
		case "", "config", "kubermaticconfiguration":
```

File: cmd/kubermatic-installer/cmd_print.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		Usage:       "Prints example configuration manifest",
+		UsageText:   "kubermatic-installer print <resource>",
+		Description: "Print an example configuration manifest with defaults for the given resource.\n   Supported resources are \"seed\" and \"kubermaticconfiguration\".",
+		ArgsUsage:   "Supported resources are \"seed\" and \"kubermaticconfiguration\".",
+		Action:      PrintAction(logger),
+	}
+}
+
+func PrintAction(logger *logrus.Logger) cli.ActionFunc {
+	return handleErrors(logger, setupLogger(logger, func(ctx *cli.Context) error {
```
Comment (kron4eg) [inline cmd/kubermatic-installer/cmd_print.go]: isn't logger is not used? 

```suggestion
	return func(ctx *cli.Context) error {
```

**Comment by mate4st** [comment]
/hold
wait for https://github.com/kubermatic/kubermatic/issues/7400

**Review by kron4eg — APPROVED** [review]
/approve

**Pull request merged**